dynamicRooms/README.md

```diff
@@ -0,0 +1,21 @@
+# RSCBot: combineRooms
```

Comment:
I assume this is meant to be dynamicRooms here instead of combineRooms

Reply:
yes

Reply:
done